index.html

```diff
@@ -149,7 +150,7 @@ <h3>Напишите нам</h3>
      <div class="popup-container">
      <label>Ваше имя:<br><input type="text" placeholder="Имя Фамилия" name="name and surname"></label>
      <label>Ваш email:<br><input type="email" placeholder="email@example.com" name="email"></label>
-     <label>Текст письма:<br><input type="textarea" placeholder="В свободной форме" name="other text"></label>
+     <label>Текст письма:<br><input class="input-other-text" type="text" placeholder="В свободной форме" name="other text"></label>
```

Comment:
[Б4](https://up.htmlacademy.ru/htmlcss/31/criteries#b4):
```suggestion
     <label>Текст письма: <input class="input-other-text" type="text" placeholder="В свободной форме" name="other text"></label>
```
Здесь может быть 2 варианта:
а) не вкладывать `input` в `label`, тогда нужно добавить обёртку для `label` и `input` и расположить элементы по сетке с помощью `flex`;
б) обёрткой может быть сама метка (как в текущем варианте), но тогда нужно будет текст метки обернуть в `span`, чтобы можно было управлять флекс-элементом.

То же самое нужно сделать и для полей "Ваше имя:", "Ваш email:". Использовать `br` для переноса HTML элемента на другую строку нельзя.